docs/group_guidelines/git_standards.md

```diff
@@ -0,0 +1,86 @@
+# Git - Standard Practices
+The purpose of this document is to provide our team with a set of standards to follow throughout our capstone project.
+With this document, we hope to ensure that we stay organized and consistent throughout the term.
+
+# Branches
+
+We will be following a technique called feature branches, where we create a new branch for every feature which is to be
+developed. This allows us to prevent merge conflicts, and closely monitor each new feature we need to create. As well,
+feature branches ensure we are dividing properly dividing issues into achievable tasks. Once the feature has been 
+completed, and the branch has been merged to main, we will NOT be deleting the branch. If bugs occur with the feature, 
+we can return to the branch in order to fix the bug. 
+
+## Naming:
+Branch names should be consistent throughout our repository. Below is the naming scheme which we should follow for all 
+feature branches: 
+
+<issue-id>_<brief-feature-description>
+
+Ex: 7_create-git-standards-documentation
+
+This naming scheme ensures that we are properly creating issues for each branch in order to ensure gitstats stays happy
```

Comment:
when we add issues for bugs, that implies we'd be creating new branches because the issue number would be different. This contradicts what's said above on lines 10-11

Reply:
I think it's ok, issues like that which are linked to other issues don't necessarily need their own branch. Branches require issues but issues don't necessarily require unique branches

Reply:
You can link multiple issues to an individual branch as well

Reply:
That seems good enough for our purposes.  Marking this conversation as resolved.